UnityProject/Assets/Scripts/Debug/TADB.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Runtime.CompilerServices;
+
+
+public class TADB_Debug
+	{
+	public enum Category{
+		Atmospherics,
+		Movement,
+		MapLoad, 
+		RoomControl,
+		ItemList,
+		ItemEntry,
+		DmiIconData,
+		UI,
+		MatrixManager,
+		Equipment,
+		NetworkManager,
+		uniCloth,
+		NetworkTabManager,
+		ItemFactory,
+		Light2D,
+		PlayerList,
+		PoolManager,
+		SpriteManager,
+		PlayerNetworkActions,
+		SteamIntegration,
+		TabUISystem,
+		Telecommunications,
+		PlayerJobManager,
+		PushPull,
+		NetSpriteImage,
+		ShutterController,
+		ShipNavigation,
+		Lighting,
+		Electrical, 
+	};
+	//-1 will not show anything
+	// 0 Only will show errors 
+	// 1 will show errors and warnings
+	// 2 will show errors, warnings and logs
+	private static Dictionary<string, int> NameDictionary = new Dictionary<string, int>{
+		["Atmospherics"] = 0,
+		["Movement"] = 0,
+		["MapLoad"] = 0,
+		["RoomControl"] = 0,
+		["ItemList"] = 0,
+		["ItemEntry"] = 0,
+		["DmiIconData"] = 0,
+		["UI"] = 0,
+		["MatrixManager"] = 0,
+		["Equipment"] = 0,
+		["NetworkManager"] = 0,
+		["uniCloth"] = 0,
+		["NetworkTabManager"] = 0,
+		["ItemFactory"] = 0,
+		["Light2D"] = 0,
+		["PlayerList"] = 0,
+		["PoolManager"] = 0,
+		["SpriteManager"] = 0,
+		["PlayerNetworkActions"] = 0,
+		["SteamIntegration"] = 0,
+		["TabUISystem"] = 0,
+		["Telecommunications"] = 0,
+		["PlayerJobManager"] = 0,
+		["PushPull"] = 0,
+		["NetSpriteImage"] = 0,
+		["ShutterController"] = 0,
+		["ShipNavigation"] = 0,
+		["Lighting"] = 0,
+		["Electrical"] = 0,
+	};
+	//Example of how to call TADB_Debug.Log("Message",TADB_Debug.Category.#Something#.ToString());
+	private static Dictionary<int, string> LevelMessage = new Dictionary<int, string>{
```

Comment:
why not use an enum for these? would be much more efficient:
public enum LogLevel {
Error = 0,
Warning = 1,
Log = 2,
Trace = 3
};
and then getting ordinal by casting to int